apps/docusaurus/docs/designsystems/Komponenter/radiobuttons.md

```diff
@@ -52,30 +56,35 @@ Radioknappene får en fokus sirkel når de velges med tastaturet.
 
 ## Feilmelding radioknapp
 
-Følgende radioknapp får feilmeldingsfarge hvis et av alternativene må være valgt før bruker kan gå videre. Merk at i dette eksempelet er <code>checked={false}</code> brukt for å tvinge frem en feilmelding som demonstrasjon.
+Følgende radioknapp får feilmeldingsfarge hvis et av alternativene må være valgt før bruker kan gå videre. Knappene skal
+ikke ha en "Valgt" dot ved feilmelding.
```

Comment:
Dette eksempelet ser litt rart ut uten javascript. I en reell situasjon skal ikke feilmeldingen vises før brukeren forsøker å gå videre, og den skal forsvinne igjen når brukeren velger en radioknapp. Dette klarer vi ikke å illustrere i dette eksempelet, og poenget er uansett å vise hvordan det ser ut når feilmeldingen vises. Men vi bør kanskje skrive noe om det? Dette er også et ganske godt case for å lage react-komponent, fordi vi da kan kode inn dette riktig i selve komponentet slik at brukerne slipper å tenke over det selv.